packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -150,7 +150,7 @@ export function blunderbuss(app: Probot) {
         // Don't check all labels to avoid updating an old issue when someone
         // changes a random label.
         const assigneesForNewLabel = findAssignees(byConfig, [
-          context.payload.label?.name,
```

Comment:
these do very different things!  The `?` will cause this to return `undefined` if `label` is undefined, but the `!` says "trust me, this will never be undefined or null".  Are you certain this is what you wanted?

Reply:
I am expecting there is a value in label given that we've waited to enter this block only if the flag `isLabeled` is true, which means that it was kicked off if the PR or issue was labeled. But, just to be on the safe side, I included an if-block instead to confirm this value exists.